ui/pages/performance/results.js

```diff
@@ -1,15 +1,14 @@
 import React, { useEffect } from 'react';
-import { NoSsr, Paper, withStyles } from '@material-ui/core';
+import { NoSsr } from '@mui/material';
+import { Paper, SistentThemeProvider } from '@layer5/sistent';
```

Comment:
not required to use Sistentthemeprovider. There is already a wrapper use that : usesSisistent

Reply:
and why we directly updating pages, are the parent component updated to sistent that is here MesheryResults? That should done first.